### PR TITLE
Release 0.2.0 candidate + additional instructions

### PR DESCRIPTION
**Pull request by ivanzvonkov**
- Moved Google Cloud to mandatory package level
- Added release instructions + zenodo script (separate file from contributing.md because one of us has to do it due to private Zenodo token and pypi username and password)
- Uploaded and tested on testpypi

### CODE REVIEW

File: release_steps.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    ```bash
+    ./zenodo_upload.sh $DEPOSITION_ID /home/ubuntu/cropharvest/data/eo_data.tar.gz
+    ./zenodo_upload.sh $DEPOSITION_ID /home/ubuntu/cropharvest/data/features.tar.gz
+    ./zenodo_upload.sh $DEPOSITION_ID /home/ubuntu/cropharvest/data/labels.geojson
+    ```
+6. Go back to the Zenodo page, refresh and verify that the new tar.gz size is updated.
+7. Click publish
+
+
+/Users/izvonkov/nasaharvest/cropharvest/dist/cropharvest-0.2.0.tar.gz
```
Comment (gabrieltseng) [inline release_steps.md:32]: Might be an accidental addition?

Comment (ivanzvonkov) [reply release_steps.md]: wow quick good eye, removed

**Review by gabrieltseng — APPROVED** [review]
LGTM - thanks @ivanzvonkov  :)

**Comment by ivanzvonkov** [comment]
Also I was doing the correct versioning on pypi, each dataset update was a micro version update (ie. 0.0.1 -> 0.0.2). I updated the Github releases to follow that here https://github.com/nasaharvest/cropharvest/releases

**Pull request merged**